src/__tests__/types.test.ts

```diff
@@ -118,6 +118,35 @@ describe('types', () => {
       });
     });
 
+    describe('decimal', () => {
```

Comment:
Please move this above, so that the sections are alphabetical: `date`, `decimal`, `enum`